<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1279</width>
    <height>714</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#mainwindow {
background: #FEFEF9
;
}


#mainFrame {
border: 3px solid gray;
border-radius: 40px;
background: white;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1271</width>
      <height>971</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#0c364e
;
color:white;</string>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="Accueil">
     <attribute name="title">
      <string>Home</string>
     </attribute>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>580</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #0f425f
  ;
color:white ;
border-style:outset;
border-width: 2px;
    border-radius: 10px;
    border-color: #0c364e;
    font:  18px;
    min-width:5em;
    padding: 6px;</string>
      </property>
      <property name="text">
       <string>Valider</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>150</y>
        <width>281</width>
        <height>78</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">
color:white ;
border-style:outset;
border-width: 0px;
    border-radius: 10px;
    border-color: #0c364e;
    font: bold 18px;
    min-width: 10em;
    padding: 6px;</string>
      </property>
      <property name="text">
       <string>Entrer Vos informations</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>20</y>
        <width>281</width>
        <height>621</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="a">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: #0c364e
  ;
color:white ;

    font:  18px;
</string>
           </property>
           <property name="text">
            <string>Accueil</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="b">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: #0c364e
  ;
color:white ;

    font:  18px;
</string>
           </property>
           <property name="text">
            <string>Gestion des vols</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="e">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: #0c364e
  ;
color:white ;

    font:  18px;
</string>
           </property>
           <property name="text">
            <string>Gestion des voyageurs</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="d">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: #0c364e
  ;
color:white ;

    font:  18px;
</string>
           </property>
           <property name="text">
            <string>Gestions des employ√©es</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="c">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: #0c364e
  ;
color:white ;

    font:  18px;
</string>
           </property>
           <property name="text">
            <string>Gestion des avions</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_pic">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>141</width>
        <height>101</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_username">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>270</y>
        <width>101</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_mdp">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>410</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>20</y>
        <width>742</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white ;
border-style:outset;
border-width: 0px;
    border-radius: 10px;
    border-color: #0c364e;

font: bold 75 25pt &quot;Segoe Print&quot;;
    min-width: 10em;
    padding: 6px;</string>
      </property>
      <property name="text">
       <string>SMART AIRPORT</string>
      </property>
     </widget>
     <widget class="QWidget" name="formLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>240</y>
        <width>751</width>
        <height>271</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">
color:white ;
border-style:outset;
border-width: 0px;
    border-radius: 10px;
    border-color: #0c364e;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;</string>
         </property>
         <property name="text">
          <string>Nom d'utilisateur:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">
color:white ;
border-style:outset;
border-width: 0px;
    border-radius: 10px;
    border-color: #0c364e;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;</string>
         </property>
         <property name="text">
          <string>Mot de passe:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Ajouter</string>
     </attribute>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>30</y>
        <width>481</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white ;
border-style:outset;
border-width: 0px;
    border-radius: 10px;
    border-color: #0c364e;
    font: bold 18px;
    min-width: 10em;
    padding: 6px;</string>
      </property>
      <property name="text">
       <string>Ajout des employ√©es </string>
      </property>
     </widget>
     <widget class="QWidget" name="formLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>671</width>
        <height>461</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_6">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">color:white ;
border-style:outset;
border-width: 0px;
    border-radius: 10px;
    border-color: #0c364e;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;</string>
         </property>
         <property name="text">
          <string>C.I.N:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">
border-style:outset;
border-width: 2px;
    border-radius: 10px;
    border-color: white;
    min-width:5em;
    padding: 6px;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_7">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">color:white ;
border-style:outset;
border-width: 0px;
    border-radius: 10px;
    border-color: #0c364e;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;</string>
         </property>
         <property name="text">
          <string>Prenom:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_8">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">color:white ;
border-style:outset;
border-width: 0px;
    border-radius: 10px;
    border-color: #0c364e;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;</string>
         </property>
         <property name="text">
          <string>Nom:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit_5">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">
border-style:outset;
border-width: 2px;
    border-radius: 10px;
    border-color: white;
    min-width:5em;
    padding: 6px;</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_10">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">color:white ;
border-style:outset;
border-width: 0px;
    border-radius: 10px;
    border-color: #0c364e;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;</string>
         </property>
         <property name="text">
          <string>mail:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lineEdit_7">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">
border-style:outset;
border-width: 2px;
    border-radius: 10px;
    border-color: white;
    min-width:5em;
    padding: 6px;</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_11">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">color:white ;
border-style:outset;
border-width: 0px;
    border-radius: 10px;
    border-color: #0c364e;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;</string>
         </property>
         <property name="text">
          <string>M√©tier:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="lineEdit_8">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">
border-style:outset;
border-width: 2px;
    border-radius: 10px;
    border-color: white;
    min-width:5em;
    padding: 6px;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">
border-style:outset;
border-width: 2px;
    border-radius: 10px;
    border-color: white;
    min-width:5em;
    padding: 6px;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>430</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #0f425f
  ;
color:white ;
border-style:outset;
border-width: 2px;
    border-radius: 10px;
    border-color: #0c364e;
    font:  18px;
    min-width:5em;
    padding: 6px;</string>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>160</y>
        <width>331</width>
        <height>181</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <zorder>label_13</zorder>
     <zorder>label_5</zorder>
     <zorder>formLayoutWidget</zorder>
     <zorder>pushButton_2</zorder>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Afficher</string>
     </attribute>
     <widget class="QTableView" name="tab_e">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>70</y>
        <width>981</width>
        <height>511</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Supprimer</string>
     </attribute>
     <widget class="QPushButton" name="pb_supp">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>310</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #0f425f
  ;
color:white ;
border-style:outset;
border-width: 2px;
    border-radius: 10px;
    border-color: #0c364e;
    font:  18px;
    min-width:5em;
    padding: 6px;</string>
      </property>
      <property name="text">
       <string>Valider</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>50</y>
        <width>301</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white ;
border-style:outset;
border-width: 0px;
    border-radius: 10px;
    border-color: #0c364e;
    font: bold 18px;
    min-width: 10em;
    padding: 6px;</string>
      </property>
      <property name="text">
       <string>Suppression d'un employ√©</string>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>200</y>
        <width>571</width>
        <height>71</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="styleSheet">
          <string notr="true">color:white ;
border-style:outset;
border-width: 0px;
    border-radius: 10px;
    border-color: #0c364e;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;</string>
         </property>
         <property name="text">
          <string>Donner le cin √† supprimer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_supp">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">
border-style:outset;
border-width: 2px;
    border-radius: 10px;
    border-color: white;
    min-width:5em;
    padding: 6px;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Modifier</string>
     </attribute>
     <widget class="QWidget" name="formLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>120</y>
        <width>581</width>
        <height>181</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>CIN:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Metier:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_6"/>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_9"/>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>410</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1279</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
